src/DatabasicCommands.java

```diff
@@ -0,0 +1,4 @@
+// https://stackoverflow.com/a/19894176
+enum DataBasicCommands{
+    insert, help
```

Comment:
Enums in Java should be `CAPITAL_SNAKE_CASE`. Although lower case will work, it can look confusing when used in code.
If you want these to be lower case when printing them, consider creating a method that makes them lower case. The best way of doing that is overriding the `toString()` method, as mentioned in the [highest voted answer](https://stackoverflow.com/a/19894094/2422278) in the stackoverflow link you mentioned.

```java
public String toString() {
    return name().toLowerCase();
}
```